apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -101,7 +101,7 @@ private static Patient transform(Beneficiary beneficiary, RequestHeaders request
 
     // Required values not directly mapped
     patient.getMeta().addProfile(ProfileConstants.C4BB_PATIENT_URL);
-    patient.setId(beneficiary.getBeneficiaryId());
+    patient.setId(String.valueOf(beneficiary.getBeneficiaryId()));
```

Comment:
Same here I guess; are we not taking the conversion the logical conclusion and adjusting dependent fields to also be long? (Or can we not do it yet until we change the table?)

Reply:
this is a FHIR object from the HAPI library and it doesn't have an overridden version that takes a long